packages/odie-client/src/components/message/messages-container.tsx

```diff
@@ -52,6 +52,12 @@ export const MessagesContainer = ( { currentUser }: ChatMessagesProps ) => {
 		return currentMessage === nextMessage;
 	};
 
+	const removeDislikeStatus = experimentVariationName === 'give_wapuu_a_chance';
+
+	const availableStatusWithFeedback = removeDislikeStatus
+		? [ 'sending', 'transfer' ]
+		: [ 'sending', 'dislike', 'transfer' ];
```

Comment:
We've talked a bit about this, but I still don't understand what the dislike status means. Why is the behavior different when the experiment is active?

Reply:
It hides the escalation when negatively rating a message:
![image](https://github.com/user-attachments/assets/e4b4f3e3-02ee-41a4-b927-934bde8f6594)

If we add the dislike to the `availableStatusWithFeedback`:
![image](https://github.com/user-attachments/assets/ab341e71-0960-47d8-90ce-44947f4908f6)

So in summary, if the user negatively adds a feedback, it will hide the message with the direct escalation button :)

So, if a message with 'dislike' status is processed, it won't add the dislike message with the button for escalating